### PR TITLE
feat(ddm-formulas): Unary operator

### PR DESCRIPTION
**Pull request by ArthurKnaus**
Allow the unary operator `-`.

Screenshot 2024-03-07 at 14 37 37

### CODE REVIEW

File: static/app/views/ddm/formulaParser/formula.pegjs
Comment (iambriccardo) [inline static/app/views/ddm/formulaParser/formula.pegjs:12]:
```suggestion
number              = [0-9]+('.'[0-9]+)? { return { type: "number", content: text()}}
```

**Comment by ArthurKnaus** [comment]
Do not merge until snuba part is merged.
https://github.com/getsentry/snuba-sdk/pull/179

**Review by obostjancic — APPROVED** [review]

**Pull request merged**
